source/fundamentals/builders.txt

```diff
@@ -8,12 +8,12 @@ Builders
 
 .. toctree::
 
-   /fundamentals/builders/aggregates
-   /fundamentals/builders/filters
-   /fundamentals/builders/indexes
-   /fundamentals/builders/projections
-   /fundamentals/builders/sort
-   /fundamentals/builders/updates
+   Aggregates </fundamentals/builders/aggregates>
+   Filters </fundamentals/builders/filters>
+   Indexes </fundamentals/builders/indexes>
+   Projections </fundamentals/builders/projections>
+   Sorts </fundamentals/builders/sort>
+   Updates </fundamentals/builders/updates>
```

Comment:
These titles feel awkward. Even though they match the name of the class and the page, I'd go with something more intuitive:

```suggestion
   Aggregation </fundamentals/builders/aggregates>
   Filters </fundamentals/builders/filters>
   Indexes </fundamentals/builders/indexes>
   Projection </fundamentals/builders/projections>
   Sort </fundamentals/builders/sort>
   Update </fundamentals/builders/updates>
```